### ISSUE TITLE
multipe route declarations for a module

### ISSUE THREAD

**Issue by jeffschwartz** [jeffschwartz/rtr#2]
Allow declaration of multiple routes for a module using the "routes" attribute:

```
<rtr-router>
    <some-routes routes='[{
        "method": "get",
        "path": "/some/path/for/get",
        "handler": "somepathHandlerForGet"
    }, {
        "method": "post",
        "path": "/some/path/for/post",
        "handler": "somepathHandlerForPost"
    }, {
    .
    .
    .
    }]'></some-routes>
</rtr-router>
```

**Comment by jeffschwartz** [comment]
Implemented in v0.2.0.

**Comment by jeffschwartz** [comment]
Implemented in v0.2.0.
